packages/cli/src/config/parseStoreConfig.ts

```diff
@@ -172,13 +189,17 @@ export type StoreUserConfig<EnumNames extends StringForUnion = StringForUnion> =
    *  - abi or user type for a single-value table (aka ECS component).
    *  - FullTableConfig object for multi-value tables (or for customizable options).
    */
-  tables: TablesConfig<EnumNames>;
+  tables: TablesConfig<StaticUserTypes, StaticUserTypes>;
```

Comment:
is this change intentional?

Reply:
yes, for now it doesn't do much of anything, but the idea is to separate all types for fields, and static types for primary keys; since I'll be adding static user types probably. I imagine I'll be changing these generics again after figuring out how exactly I want to use them